### PR TITLE
workflows/deploy-manual: set branch name in auto-PR title and body

### PR DESCRIPTION
**Pull request by estroz**
**Description of the change:**
- .github/workflows/deploy-manual.yml: set branch name in auto-PR title and body

**Motivation for the change:** make PR title and commit descriptive.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: .github/workflows/deploy-manual.yml [hunk 1]
```diff
@@ -55,16 +55,19 @@ jobs:
     # This change will be staged and committed in the PR pushed below.
     # The script below will fail if no change was made.
     - name: update ansible-operator base
+      id: update
       run: |
-        set -e
```
Comment (jberkhahn) [inline .github/workflows/deploy-manual.yml:59]: TIL. this seems nice

**Comment by jberkhahn** [comment]
seems good
/lgtm

**Review by fabianvf — APPROVED** [review]
/lgtm

**Review by marc-obrien — COMMENTED** [review]
/lgtm

**Pull request merged**